iceoryx_posh/include/iceoryx_posh/internal/popo/building_blocks/chunk_queue_data.hpp

```diff
@@ -37,14 +38,15 @@ struct ChunkQueueData : public LockingPolicy
     using LockGuard_t = std::lock_guard<const ThisType_t>;
     using ChunkQueueDataProperties_t = ChunkQueueDataProperties;
 
-    explicit ChunkQueueData(cxx::VariantQueueTypes queueType) noexcept;
+    ChunkQueueData(const QueueFullPolicy policy, const cxx::VariantQueueTypes queueType) noexcept;
```

Comment:
This does not make sense if you have more than one argument like in this case here.
Explicit prevents auto conversion since the one argument ctor is a conversion ctor - this is not the case here.

Reply:
Please remove it for `iox::popo::ChunkDistributorData::ChunkDistributorData()`, too.